Capfile

```diff
@@ -0,0 +1,31 @@
+###########################################
+# Capistrano Defaults
+########################################### 
+
+# Load DSL and set up stages
+require "capistrano/setup"
+
+# Include default deployment tasks
+require "capistrano/deploy"
+
+require "capistrano/scm/git"
+install_plugin Capistrano::SCM::Git
+
+
+###########################################
+# Chapman Blogs Custom Config
+# https://github.com/chapmanu/blogs
+###########################################   
+
+# Custom Helpers
+require './lib/helpers.rb'
+
+# Include custom strategy for deploying git submodules
+require './lib/capistrano/submodule_strategy'
+
+# Enter Sudo password for restarting nginx/php
+require 'sshkit/sudo'
```

Comment:
Ok. Give yourself to end of day tomorrow to see if you can get deployments to be passwordless. If you can't find a way to do it, add TODO comments identifying what we'd need to address to accomplish this in future.